docs/index.rst

```diff
@@ -64,7 +64,7 @@ The development version can be obtained and installed from github:
 Requirements
 ------------
 
-Astroquery works with Python 2.7 and 3.5 or later.
+Astroquery works with Python 3.5 or later.
```

Comment:
```suggestion
Astroquery works with Python 3.5 or later.  Some modules may be compatible with python 2.7, but support for python 2 is not provided.
```

Reply:
I prefer to note that it still does work with python 2 - we haven't explicitly broken everything (yet)

Reply:
Yeah, but folks can always try, I just find it cleaner not to give the slightest promise for it explicitly. 

Also, at some point we need to clean up.

Reply:
Why not straight to Python 3.6 and then use f-string? 😏

Reply:
Yes, I would be on board with that, too.

Reply:
Me too, I don't have any ties to py35 at all.

But, the above statement will certainly trigger problems I never thought to consider on innumerable projects, so...